src/user-icon/user-icon.scss

```diff
@@ -23,13 +23,15 @@
     border-radius: 50%;
     box-sizing: border-box;
     @include themed {
-      box-shadow: inset 0 0 0 2px
+      box-shadow: inset 0 0 0 $iui-xxs
         rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
     }
   }
 
   > .iui-initials {
-    display: block;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
You can use `place-content: center`

Reply:
Are you sure?  I'm getting this after replacing line 34:
![Screen Shot 2021-06-18 at 2 36 41 PM](https://user-images.githubusercontent.com/849817/122603610-9b7cd300-d042-11eb-97b3-28eb3a1fbd76.png)

Reply:
Ah nevermind, I thought you used `justify-content` and `align-content` (not `align-items`).